orienteer-pivottable/src/main/java/org/orienteer/pivottable/PivotTableModule.java

```diff
@@ -19,15 +19,19 @@
 	public static final String NAME = "pivottable";
 	public static final String WIDGET_OCLASS_NAME = "PivotTableWidget";
 	public static final String OPROPERTY_PIVOT_TABLE_CONFIG = "pivotTableConfiguration";
+	public static final String OPROPERTY_PIVOT_CUSTOM_SQL = "sql";
 	
 	protected PivotTableModule() {
-		super(NAME, 1, OWidgetsModule.NAME);
+		super(NAME, 2, OWidgetsModule.NAME);
 		
 	}
 	
 	@Override
 	public ODocument onInstall(OrienteerWebApplication app, ODatabaseDocument db) {
 		onUpdate(app, db, 0, getVersion());
+		OSchemaHelper helper = OSchemaHelper.bind(db);
```

Comment:
Why it's here? Please try to understand what onInstall and onUpdate are doing. If keep as you wrote: it will be a duplication.